packages/grid/grid.css

```diff
@@ -0,0 +1,47 @@
+:root {
+  /* Settings */
+  --grid-columns-number: 24;
+  --grid-columns-min: 13;
+  --grid-columns-max: 50;
+  --grid-width-max: 1920;
+  --grid-column-gap: 0;
+  --grid-margin-inline: 2vw; /* TODO: 'vw' should be replaced with 'vi' following the support of the browsers */
+
+  /* Values calculation & Units assignment */
+  --grid-width-min: calc(var(--grid-columns-number) * var(--grid-columns-min));
+  --grid-width-min-size: calc(var(--grid-width-min) * 1px);
+  --grid-width-max-size: calc(var(--grid-width-max) * 1px);
+  --grid-columns-min-size: calc(var(--grid-columns-min) * 1px);
+  --grid-columns-max-size: calc(var(--grid-columns-max) * 1px);
+  --grid-column-gap-size: calc(var(--grid-column-gap) * 1px);
+  --grid-column-size: calc(100% / var(--grid-columns-number));
+
+  /* Layout */
+  --grid-columns-minmax: minmax(var(--grid-columns-min-size), var(--grid-columns-max-size));
+  --grid-template-columns-number: repeat(var(--grid-columns-number), var(--grid-columns-minmax));
+  --grid-template-columns: [expand-start] minmax(var(--grid-margin-inline), 1fr) [grid-start] var(--grid-template-columns-number) [grid-end] minmax(var(--grid-margin-inline), 1fr) [expand-end];
+}
+
+.grid {
+  min-width: var(--grid-width-min-size);
+  max-width: var(--grid-width-max-size);
+  display: grid;
+  grid-template-columns: var(--grid-template-columns);
+  grid-template-areas: var(--grid-template-areas);
```

Comment:
Does it use a variable that is not declared anywhere?

Reply:
Yep. It's provided an empty variable with a unified namespace. Something not OK with it?

Reply:
Removed.